content/en/users/security/secrets-store/cli/_index.md

```diff
@@ -0,0 +1,722 @@
+       redis_password=abcdef
+```
+
+{{< /tabx >}} {{< tabx header="Windows" >}}
+
+```shell
+> fedcloud secret put my_app_secrets ^
+       mysql_password=123456 ^
+       redis_password=abcdef
+```
+
+{{< /tabx >}}{{< /tabpanex >}}
+
+Listing all your secrets is very simple:
+
+```shell
+$ fedcloud secret list
+my_app_secrets
+```
+
+Using any of the keys (actually, their vaules) is straighforward too:
+
+{{< tabpanex >}} {{< tabx header="Mac / Linux" >}}
+
+```shell
+# Get all keys and their values
+$ fedcloud secret get my_app_secrets
+key             value
+--------------  -------
+redis_password  abcdef
+mysql_password  123456
+
+# Get the value of a specific key from a secret
+$ fedcloud secret get my_app_secrets mysql_password
+123456
+
+# Using the value of a specific key from a secret
+$ export ADMIN_PASSWORD=$(fedcloud secret get my_app_secrets admin_password)
+$ echo $ADMIN_PASSWORD
+abcdef
+```
+
+{{< /tabx >}} {{< tabx header="Powershell" >}}
+
+```powershell
+# Get all keys and their values
+> fedcloud secret get my_app_secrets
+key             value
+--------------  -------
+redis_password  abcdef
+mysql_password  123456
+
+# Get the value of a specific key from a secret
+> fedcloud secret get my_app_secrets mysql_password
+123456
+
+# Using the value of a specific key from a secret
+> $env:ADMIN_PASSWORD=$(fedcloud secret get my_app_secrets admin_password)
+> echo $env:ADMIN_PASSWORD
+abcdef
+```
+
+{{< /tabx >}} {{< tabx header="Windows" >}}
+
+```shell
+:: Get all keys and their values
+> fedcloud secret get my_app_secrets
+key             value
+--------------  -------
+redis_password  abcdef
+mysql_password  123456
+
+:: Get the value of a specific key from a secret
+> fedcloud secret get my_app_secrets mysql_password
+123456
+
+:: Using the value of a specific key from a secret
+> set ADMIN_PASSWORD=(fedcloud secret get my_app_secrets admin_password)
+> echo %ADMIN_PASSWORD%
+abcdef
+```
+
+{{< /tabx >}}{{< /tabpanex >}}
+
+Deleting a secret is easy, but it is irreversible:
+
+```shell
+$ fedcloud secret delete my_app_secrets
+```
+
+### Secret values from small text files
+
+If the secret value starts with `@` the content of the file with that name is
+used as the value of the key. The following example creates a secret named
+`certificate` for storing the certificate file and its keyfile:
+
+```shell
+$ fedcloud secret put certificate cert=@hostcert.pem key=@hostkey.pem
+```
+
+You can get the certificate and its keyfile (e.g. when you want to use it on a
+virtual machine) as follows:
+
+```shell
+$ fedcloud secret get certificate cert > hostcert.pem
+$ fedcloud secret get certificate key  > hostkey.pem
+```
+
+{{% alert title="Note" color="info" %}} The size of the secret object (all the
+values in it) is limited to 512kB, which is sufficient for storing tokens,
+certificates, configuration files and so on. For larger datasets, please use
+permanent [cloud storage](../../../data/storage).{{% /alert %}}
```

Comment:
> For larger datasets [...]

Are we still talking about secrets? IMO I wouldn't mix the secrets storage with normal storage.

Reply:
Yes, we do. But if it's larger than half a megabyte, you need a different solution. Maybe some encrypted storage.

Reply:
In that case I would explicitly mention that the permanent storage offered is not encrypted by default.

I believe we only provide this?

https://docs.egi.eu/users/data/storage/block-storage/#storage-encryption

which is only a subset of https://docs.egi.eu/users/data/storage/

Just trying to be as specific as possible.

Reply:
Updated link